appeals/web/src/server/views/appeals/appeal/schedule-site-visit.njk

```diff
@@ -79,17 +79,17 @@
 				{
 					value: "unaccompanied",
 					text: "Unaccompanied",
-					checked: siteVisit.visitType === "unaccompanied"
+					checked: (siteVisit.visitType === "unaccompanied" or siteVisit.visitType === "Unaccompanied")
```

Comment:
If this is to handle values from the API response, it should probably be done in the mapper instead

Reply:
I think the best thing to do would be to fix the API so the Get (Unaccompanied) and Post (unaccompanied) are the same value formats. I'll create a ticket to address that and put a note to remove redundant code after it has been done. Would that be sufficient or should I build a temporary solution in the mapper too?

Reply:
I agree this should be fixed API side ideally. In principle though there is still mapping occurring directly in the view here, strictly speaking it should be moved into the mapper regardless of what the API values are